test/run_test.py

```diff
@@ -0,0 +1,221 @@
+    package = package[:package.rfind(".")]
+    return (corepath, package, sys.executable)
+
+#create local config file, change what is needed and edit env variable
+def create_local_config(change_repo_dir):
+    #create local copy of set configuration file
+    #read current configuration file
+    config_path = os.environ.get('RETRACE_SERVER_CONFIG_PATH')
+    new_conf_path = '/var/tmp/retrace-server.conf'
+    with open(config_path) as f:
+        lines = f.read().splitlines()
+
+    #if not dont_create_repo arg was specified, change RepoDir to /var/tmp
+    if not change_repo_dir:
+        repo_dir_old = [x for x in lines if x.startswith("RepoDir")][0]
+        repo_dir_new = 'RepoDir = /var/tmp'
+        lines = [x if x != repo_dir_old else repo_dir_new for x in lines]
+
+    #change RequireGPGCheck to 0
+    gpg_old = [x for x in lines if x.startswith("RequireGPGCheck")][0]
+    gpg_new = 'RequireGPGCheck = 0'
+    lines = [x if x != gpg_old else gpg_new for x in lines]
+
+    #write new config file
+    with open(new_conf_path, "w") as f:
+        for line in lines:
+            f.write("%s\n" % line)
+
+    #edit env variable RETRACE_SERVER_CONFIG_PATH
+    os.environ["RETRACE_SERVER_CONFIG_PATH"] = new_conf_path
+
+#Parse arguments
+parser = argparse.ArgumentParser(description="Run test for retrace-server.")
+parser.add_argument("--coredump", help="path to the coredump, if not specified,\
+                                        will be generated")
+parser.add_argument("--release", help="os_release,\
+                                      default=Fedora release 24 (Twenty Four)")
+parser.add_argument("--executable", help="path to the executable")
+parser.add_argument("--package", help="name of package")
+parser.add_argument("--delete_repo", help="remove repo after bt generation",
+                                     action="store_true")
+parser.add_argument("--dont_create_repo", help="don't create repo before bt\
+                                                generation",
+                                                action="store_true")
+args = parser.parse_args()
+
+#set default values
+os_release = args.release if args.release else "Fedora release 24 (Twenty Four)"
+dont_create_repo_arg = True if args.dont_create_repo else False
+delete_repo_arg = True if args.delete_repo else False
+
+#create edited coy of configuration file
+create_local_config(dont_create_repo_arg)
+
+#if coredump specified, parse packages and executable
+if args.coredump:
+    coredump_path = args.coredump
+    package = args.package
+    executable = args.executable
+    if not package or not executable:
+        print "ERROR: coredump specified, must specify package and executable"
+        sys.exit(1)
+#if not coredump specified, generate it
+else:
+    coredump_path, package, executable = generate_coredump()
+
+#edit PATH so coredump2packages will be found
+cwd = os.getcwd()
+os.environ["PATH"] += os.pathsep +  cwd[:cwd.rfind("/")+1]+'src/'
```

Comment:
Please, update the PATH environment variable outside the script. Don't forget to prepend your changes ;)